extensions/amp-analytics/0.1/vendors.js

```diff
@@ -1229,6 +1229,33 @@ export const ANALYTICS_CONFIG = /** @type {!JSONType} */ ({
       'image': true,
     },
   },
+
+  'rakam': {
+    'vars': {
+      'deviceId': 'CLIENT_ID(rakam_device_id)',
+    },
+    'requests': {
+      'base': '?api.api_key=${writeKey}' +
+        '&prop._platform=amp' +
+        '&prop._device_id=${deviceId}' +
+        '&prop.locale=${browserLanguage}' +
+        '&prop.path=${canonicalPath}' +
+        '&prop.url=${canonicalUrl}' +
+        '&prop.color_depth=${screenColorDepth}' +
+        '&prop._referrer=${documentReferrer}' +
+        '&prop.title=${title}' +
+        '&prop.timezone=${timezone}' +
+        '&prop._time=${timestamp}' +
+        '&prop.resolution=${screenWidth} × ${screenHeight}',
+      'pageview': 'https://${apiEndpoint}/event/pixel${base}&collection=${pageViewName}',
+      'custom': 'https://${apiEndpoint}/event/pixel${base}&collection=${collection}',
+    },
+    'transport': {
+      'beacon': false,
```

Comment:
in case this is a copy from other config. 
`beacon: true` is recommended if your server does support it.